### DISCUSSION TITLE
How does an AI driven model deal with changes in ownership of a home?

### DISCUSSION THREAD

**Discussion by esc-data-science** [alan-turing-institute/ADViCE#97] [Key sector: Built Environment]
How does the AI driven model, based on specific customer data, deal with changes in ownership, or even changes within the circumstances within a home?  Could subsequent owners find that the system is inadequate because their needs are different?  Is there more wisdom in designing a system based on the parameters of the home rather than the consumer, but with a smart system which is able to adapt to occupier behaviour?

**Comment by samyoung-dsci** [comment]
This is a great point that really needs to be taken into account in the AI system design. In theory if behaviour changes are significant then they should be visible in the data and the system should be able to adapt, but the algorithm needs to be designed to do that (and to do it quickly enough). The risk is probably greater for some types of homes (e.g. rental properties with shorter tentures) than others. 
 
A way to think about this is as a trade-off between 'optimal in the short term' and 'resilient to changes' - that's a fundamental challenge in optimisation of most things.

**Comment by samyoung-dsci** [comment]
[Steve]: I think the consumers internally will always be a large component of the energy demand, although I believe there be some links to the house type.

So I see it as advocating for adaptive models which update strongly based on new methods. However, methods like transfer learning can be the basis for initial priors for models, which can be fine tuned as data is observed from the new occupants. How much data is required for a sufficient performance is one question
